ui/src/Components/notification/index.js

```diff
@@ -0,0 +1,32 @@
+import { NOTIFICATION_TEXT_COLOR } from "../../Constant/constant";
+import { useNotificationContext } from "../../Context/NotificationContext";
+
+const NotificationParent = () => {
+  const { notificationQueue } = useNotificationContext();
+
+  return (
+    <div
+      data-testid="notificationparent"
+      className="flex justify-center items-center flex-col-reverse fixed top-4 w-max gap-2 transition-[height] duration-300 ease-[cubic-bezier(0.165, 0.84, 0.44, 1)]"
+      //this piece of code will increase or decrease the height of the wrapper accr to the no. of notifications present
+      style={{ height: `${notificationQueue.length * 2.5}rem` }}
```

Comment:
Tailwind's arbitrary values can be used here
https://tailwindcss.com/docs/height#arbitrary-values

Reply:
Also, do we need to explicitly increase the height of this container. Won't it grow based on the content inside it?